src/app/shared/state/workflow.query.ts

```diff
@@ -26,10 +26,12 @@ export class WorkflowQuery extends QueryEntity<WorkflowState, Service | BioWorkf
     )
   );
   public extendedDescriptorLanguageBean$: Observable<ExtendedDescriptorLanguageBean> = this.workflow$.pipe(
-    map((workflow) => this.descriptorLanguageService.workflowDescriptorTypeEnumToExtendedDescriptorLanguageBean(workflow.descriptorType))
+    map((workflow) =>
+      workflow ? this.descriptorLanguageService.workflowDescriptorTypeEnumToExtendedDescriptorLanguageBean(workflow.descriptorType) : null
+    )
   );
   public launchSupport$: Observable<boolean> = this.extendedDescriptorLanguageBean$.pipe(
-    map((extendedDescriptorLanguage) => extendedDescriptorLanguage.workflowLaunchSupport)
+    map((extendedDescriptorLanguage) => (extendedDescriptorLanguage ? extendedDescriptorLanguage.workflowLaunchSupport : null))
```

Comment:
No need to change, but for future reference, if we're ok with this expression possibly evaluating to `undefined`, we could the "optional chaining" operator here:
```
extendedDescriptorLanguage?.workflowLaunchSupport
```
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining